blockbase/sass/blocks/_post-comments.scss

```diff
@@ -1,5 +1,7 @@
 .wp-block-post-comments {
-	label, .comment-meta {
+
+	label,
```

Comment:
I don't think there are any changes in this file other than formatting.  Can we remove this change to avoid confusion?

Reply:
Oh!  Nevermind.  I found the change.  The lots'a formatting changes made it hard to find.  Nevermind this comment!